sampler/README.md

```diff
@@ -471,7 +473,7 @@ $$
 p(\mathbf{x}; \mathbf{p}) = \frac{\Gamma((\sum_{i=1}^k x_i)+1)}{\Gamma(x_1+1)\cdots{}\Gamma(x_k+1)}p_1^{x_1}\cdots{}p_k^{x_k},
 $$
 
-#### Working in "log-space"
+#### Working in "Log-Space"
```

Comment:
I've taken out references to "log-space" and done some minor rewriting to accommodate them. (Sometimes tricky -- the metaphor of being "in" log-space is useful!) Can you look over this section to check that it reads smoothly and I haven't mangled your meaning?